pkg/cmd/config/config.go

```diff
@@ -8,19 +8,31 @@ import (
 	"github.com/spf13/cobra"
 )
 
+type ConfigKey struct {
```

Comment:
Minor: I would say that this type need not be exported

Reply:
Cool, I will change it (I realised it while coding, but thought we were not exactly following the exported-unexported convention, so did not unexport it :sweat_smile: )